src/js/components/Layer/doc.js

```diff
@@ -146,6 +148,18 @@ export const themeDoc = {
     type: 'string',
     defaultValue: 'white',
   },
+  'layer.border.radius': {
+    description: 'The rounding of the Layer corners.',
+    type: 'string',
+    defaultValue: 'white',
+  },
+  'layer.border.intelligentRounding': {
+    description: `Whether the border-radius of the Layer should adapt based on 
+    the Layer's position. Wherever the Layer is touching the edge of the 
+    screen, a border-radius of 0 will be applied.`,
+    type: 'boolean',
+    defaultValue: undefined,
```

Comment:
Given that `layer.border.radius` is new, should the default value for this key should be `true`, given that's the more likely desire?

Reply:
layer.border.radius is not new, it just wasn't documented prior

Reply:
k